triggers/service/auth/src/main/scala/com/daml/oauth/middleware/README.md

```diff
@@ -18,6 +18,7 @@ repository](https://github.com/digital-asset/ex-secure-daml-infra).
   - Provide a name (`ex-daml-api`).
   - Provide an Identifier (`https://daml.com/ledger-api`).
   - Select Signing Algorithm of `RS256`.
+  - Allow offline access to enable refresh token generation.
```

Comment:
It took me way too long to understand the name here so let me record this for the future: This allows your oauth client (middleware) to request access (by virtue of a refresh token) when the resource owner (the end user) is offline.

Reply:
```suggestion
  - Allow offline access to enable refresh token generation.
    This allows the OAuth2 client, i.e. the auth middleware, to request access through a refresh token when the resource owner, i.e. the user, is offline. 
```
Good point. I'll add a sentence to clarify the meaning of the setting.